### DISCUSSION TITLE
Is it worth considering replacing `vitepress` with `starlight`?

### DISCUSSION THREAD

**Discussion by shlroland** [prosekit/prosekit#743] [General]
This project seems to lean towards using the `Astro` toolchain. Should we consider transforming the website from `vitepress` to `Astro` + [`Starlight`](https://starlight.astro.build/#:~:text=Astro%20%2B%20Starlight) project?

**Comment by ocavue** [comment]
I did consider different tools. In my opinion Vitepress has a better default visual design so I'm using it at this moment.

**Comment by shlroland** [comment]
Now it's already astro.
